app/Http/Controllers/Payments/PaypalController.php

```diff
@@ -66,6 +66,8 @@ public function create()
         $order = Order::where('user_id', Auth::user()->user_id)->processing()->findOrFail($orderId);
         $command = new PaypalCreatePayment($order);
         $link = $command->getApprovalLink();
+        // getId() is only available after the payment is created which getApprovalLink() calls.
+        $order->update(['tracking_code' => $command->getPayment()->getId()]);
```

Comment:
wouldn't this be shown to the user if the order status changes to shipped/delivered?

Reply:
I put it there because I didn't want to have to add another column to store this but looks like it needs another column 😕